packages/components/tooltip/src/tooltip.stories.ts

```diff
@@ -115,6 +116,30 @@ export const Shared: Story = {
   }
 };
 
+export const NestedChildren: Story = {
+  args: {
+    example: ({ message }) => html`
+      <div
```

Comment:
That `div` is not focusable, so it should not have a tooltip, right? Or maybe make it focusable?